### PR TITLE
Features/site status managment

### PR DESCRIPTION
**Pull request by HelkaPL**
Ok, that's my idea for handlind status. ( right now implemented for PopularPeople )

Let me know is that a good concept or we should re-think it?

### CODE REVIEW

File: src/features/people/PopularPeoplePage/index.js [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
-                  poster={person.profile_path}
-                  name={person.name}
-                />
-              </li>
-            ))}
-        </GridList>
-      </TilesSection>
-    </Container>
-  );
+  return {
```
Comment (marcin10lw) [inline src/features/people/PopularPeoplePage/index.js:11]: I love the way you did this. You'll have to tell me more about it

File: src/features/people/PopularPeoplePage/PopularPeople/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { GridList } from "../../../../common/GridList";
+
+const PopularPeople = () => {
+  const people = useSelector(selectPeople);
+
+  return (
+    <Container>
+      <TilesSection title="Popular people">
+        <GridList popularPeople>
+          {people &&
```
Comment (marcin10lw) [inline src/features/people/PopularPeoplePage/PopularPeople/index.js:15]: with loading handled we can change `people && people.map...` to just `people.map...`

**Review by marcin10lw — APPROVED** [review]
Great job, well have to think about placement of Main (the distance between error screen and Header if too high because of Main's padding)

**Review by mikolajgraczyk — APPROVED** [review]
Very nice, I approve 🤠

**Pull request merged**